app/code/Magento/Contact/Model/ContactForm.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+namespace Magento\Contact\Model;
+
+use Magento\Framework\Api\AbstractSimpleObject;
+use Magento\Contact\Api\Data\ContactFormInterface;
+
+class ContactForm extends AbstractSimpleObject implements ContactFormInterface
```

Comment:
Why do we need to extend `AbstractSimpleObject`? It could be more simple to use objects with `__construct`